docs/src/css/custom.css

```diff
@@ -257,18 +263,25 @@ strong {
   --docsearch-hit-background: var(--ifm-color-emphasis-100);
   /* Footer */
   --docsearch-footer-background: var(--ifm-background-surface-color);
-  --docsearch-key-gradient: linear-gradient(-26.5deg,
-      var(--ifm-color-emphasis-200) 0%,
-      var(--ifm-color-emphasis-100) 100%);
+  --docsearch-key-gradient: linear-gradient(
+    -26.5deg,
+    var(--ifm-color-emphasis-200) 0%,
+    var(--ifm-color-emphasis-100) 100%
+  );
 }
 
 .DocSearch-Button {
   border-radius: 5px !important;
 }
 
+.DocSearch-Search-Icon {
+  height: 13px;
+  width: 13px;
+}
```

Comment:
Only this change is required. Please remove all other changes from this PR. (Formatting and other changes)

Reply:
I have removed the unnecessary changes.